libraries/AP_Mount/AP_Mount_SoloGimbal.cpp

```diff
@@ -115,15 +115,31 @@ void AP_Mount_SoloGimbal::set_mode(enum MAV_MOUNT_MODE mode)
     _mode = mode;
 }
 
-// send_mount_status - called to allow mounts to send their status to GCS using the MOUNT_STATUS message
-void AP_Mount_SoloGimbal::send_mount_status(mavlink_channel_t chan)
+// send a GIMBAL_DEVICE_ATTITUDE_STATUS message to GCS
+void AP_Mount_SoloGimbal::send_gimbal_device_attitude_status(mavlink_channel_t chan)
 {
-    if (_gimbal.aligned()) {
-        mavlink_msg_mount_status_send(chan, 0, 0, degrees(_angle_rad.roll)*100, degrees(_angle_rad.pitch)*100, degrees(get_bf_yaw_angle(_angle_rad))*100, _mode);
-    }
-
     // block heartbeat from transmitting to the GCS
     GCS_MAVLINK::disable_channel_routing(chan);
+
+    if (!_gimbal.aligned()) {
+        return;
+    }
+
+    // construct quaternion from angle targets
```

Comment:
could this be common code in the backend?

Reply:
yes, txs.  moved to common code in the backend.